web/src/components/event/EventDetail.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import { Card } from "@blueprintjs/core";
+import { GroupEvent } from "./event-demo";
+import { withRouter, RouteComponentProps } from "react-router-dom";
+import moment from "moment";
+
+import "./EventDetail.scss";
+
+type EventDetailProps = RouteComponentProps<any> & {
```

Comment:
@alanreid I think this looks better:
```
interface EventDetailProps extends RouteComponentProps {
  event: GroupEvent | null;
};

class EventDetail extends Component<EventDetailProps> {
```

Reply:
Yeah, although apparently composition is preferred over inheritance in the React world. 
https://reactjs.org/docs/composition-vs-inheritance.html

I don't really find any issues with your code though!

Reply:
I know, but they are talking about the components and I agree. We are talking about the TypeScript interfaces.